### DISCUSSION TITLE
What's with the prohibition on ignoring the .idea directory?

### DISCUSSION THREAD

**Discussion by clink-aaron** [SocialiteProviders/Providers#806] [General]
I tried sneaking a PR through adding `.idea` to the `.gitignore`, it was removed by maintainers.

It's not just this project though, I've found it seems to be a fairly common sentiment on other laravel projects.

As many of us are using phpstorm and other jetbrains products, this seems to be a no-brainer. it's an automatic entry in all my php projects. For those of us using CLI helpers it's exceptionally annoying to see there is untracked files, and run `git status` to ensure there isn't anything uncommitted that may have been missed. 

I feel someone's sense of aesthetic is getting in the way of utility here. 

TL;DR: let's use the `.gitignore`, that's what it's there for.

**Comment by atymic** [comment]
Add it to your global gitignore, save adding it to every PHP project :)
https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/

**Comment by clink-aaron** [comment]
I suppose that's a better solution. How do I delete dis discussion? I can't find an option for doing so.

**Reply by atymic** [reply]
It's fine, leave it in case others ask :)
